[2024-09-03T17:19:33.541+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-03T17:19:33.560+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: python_operator_dag_v1.first_task manual__2024-09-03T17:19:30.083863+00:00 [queued]>
[2024-09-03T17:19:33.571+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: python_operator_dag_v1.first_task manual__2024-09-03T17:19:30.083863+00:00 [queued]>
[2024-09-03T17:19:33.573+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 3
[2024-09-03T17:19:33.633+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): first_task> on 2024-09-03 17:19:30.083863+00:00
[2024-09-03T17:19:33.683+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'python_operator_dag_v1', 'first_task', 'manual__2024-09-03T17:19:30.083863+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmp94byko75']
[2024-09-03T17:19:33.696+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=18547) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-03T17:19:33.703+0000] {standard_task_runner.py:105} INFO - Job 58: Subtask first_task
[2024-09-03T17:19:33.707+0000] {standard_task_runner.py:72} INFO - Started process 18548 to run task
[2024-09-03T17:19:33.809+0000] {task_command.py:467} INFO - Running <TaskInstance: python_operator_dag_v1.first_task manual__2024-09-03T17:19:30.083863+00:00 [running]> on host cedbab6b7256
[2024-09-03T17:19:33.942+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Abhay' AIRFLOW_CTX_DAG_ID='python_operator_dag_v1' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-09-03T17:19:30.083863+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-03T17:19:30.083863+00:00'
[2024-09-03T17:19:33.947+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-03T17:19:34.012+0000] {logging_mixin.py:190} INFO - The sum is101
[2024-09-03T17:19:34.014+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-03T17:19:34.023+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-03T17:19:34.025+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=python_operator_dag_v1, task_id=first_task, run_id=manual__2024-09-03T17:19:30.083863+00:00, execution_date=20240903T171930, start_date=20240903T171933, end_date=20240903T171934
[2024-09-03T17:19:34.100+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-03T17:19:34.160+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-03T17:19:34.168+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
